news/q2-2024-recap-from-rebecca-rumbul.md

```diff
@@ -0,0 +1,80 @@
+
+## Q2 Goals
+
+During the April 2024 Board Meeting, we set forth several ambitious goals for the second quarter. These included completing the recruitment and onboarding of three new staff members, opening the 2024 Fellowship applications, and attending key industry events such as the OSS Summit North America, DAC San Francisco, RustNL, and the SOSS EU in Brussels.
+
+The Foundation also aimed to organize a safety-critical industry forum, make further progress on training program content development, and launch its new website, although the latter two have been delayed.
+
+## HR & Administration
+
+The second quarter saw significant progress in HR and administration. [Christina Gorton](https://foundation.rust-lang.org/news/welcoming-marketing-program-manager-christina-gorton-to-the-rust-foundation-team/) joined the Foundation as the new Marketing Program Manager on April 22 and has already made a notable impact. She is actively supporting the delivery of RustConf, scaling marketing and membership activities, and initiating the development of a training program. The Foundation also completed the recruitment process for two other key roles: an Infrastructure Engineer to support JD Nose and an Interoperability Engineer to lead the Rust/C++ interop initiative. [Jon Bauman](https://foundation.rust-lang.org/news/welcoming-rust-c-interoperability-engineer-jon-bauman-to-the-rust-foundation-team/), who started on May 31, has begun orienting himself in the Rust/C++ landscape and is set to present a strategy proposal in the coming weeks. [Marco Ieni](https://foundation.rust-lang.org/news/welcoming-infrastructure-engineer-marco-ieni-to-the-rust-foundation-team/), joined the Rust Foundation on July 15 as its second Infrastructure Engineer, reporting to JD. Marco will provide much-needed support to the Rust project and its infrastructure.
+
+## Finance & Legal
+
+The Foundation transitioned its financial management and reporting systems from QuickBooks to Xero, introducing more granular financial reporting and tracking. This shift aims to better reflect how resources are allocated in support of the [Rust Project](https://www.rust-lang.org/). The Foundation completed the reporting for its 2023 990 form submission and continues to invest surplus funds into interest-bearing certificates of deposit to maximize returns. A review of performance against the 2023 budget was conducted, showing a year-to-date effective operating surplus of $139,600.
+
+## Membership
+
+In Q2, we were pleased to welcome several new members to the Rust Foundation. Wyeworks LLC, Accelerant Limited, and Zed Industries joined as new Silver Members and new Associate Members: Stichting Rust Nederland (RustNL) and the Brazilian Pontifícia Universidade Católica do Paraná.
```

Comment:
An Announcement of Wyeworks, Accelerant, and Zed is currently in the works.

Reply:
@graciegregory what line should this change go on? At the end of the membership paragraph?